app/reducers/app-data.js

```diff
@@ -1,30 +1,24 @@
 const defaultState = {
     isFetching: false,
-    users: []
+    userData: {}
 };
 
 export default function appData(state = defaultState, action) {
     switch (action.type) {
         case 'FETCHING_DATA':
             return {
                 isFetching: true,
-                users: state.users
+                userData: state.userData[0]
```

Comment:
you changed the userData to an object in the state. I would assume this will throw an error.